Students/Babisca - Fisteag Iosua - Paul/inventoryManagement/src/Customer.java

```diff
@@ -0,0 +1,26 @@
+import java.util.ArrayList;
+
+public class Customer {
+    private Integer customerID;
+    private String customerName;
+    private String customerContact;
```

Comment:
ca si denumire ajunge daca pui fara `customer`.. ca deja ai denumirea in numele clasei. Cand o sa folosesti clasa de cele mai multe ori o sa fie `Customer customer = new Customer();` si atunci e redundant sa ai `customer.getCustomerName()` vs `customer.getName()`